src/lib/schema/Schema.ts

```diff
@@ -0,0 +1,246 @@
+
+		// If the returned value is a SchemaFolder, return its result from SchemaFolder#get using remaining string
+		if (value.type === 'Folder') return (value as SchemaFolder).get(path.substring(index + 1));
+
+		// Return value
+		return value;
+	}
+
+	/**
+	 * Returns a new Iterator object that contains the keys for each element contained in this folder.
+	 * Identical to [Map.keys()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/keys)
+	 * @param recursive Whether the iteration should be recursive
+	 */
+	public *keys(recursive = false): IterableIterator<string> {
+		if (recursive) {
+			for (const [key, value] of super.entries()) {
+				if (value.type === 'Folder') yield* (value as SchemaFolder).keys(true);
+				else yield key;
+			}
+		} else {
+			yield* super.keys();
+		}
+	}
+
+	/**
+	 * Returns a new Iterator object that contains the values for each element contained in this folder and children folders.
+	 * Identical to [Map.values()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/values)
+	 * @param recursive Whether the iteration should be recursive
+	 */
+	public values(recursive: true): IterableIterator<SchemaEntry>;
+	/**
+	 * Returns a new Iterator object that contains the values for each element contained in this folder.
+	 * Identical to [Map.values()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/values)
+	 * @param recursive Whether the iteration should be recursive
+	 */
+	public values(recursive?: false): IterableIterator<SchemaFolder | SchemaEntry>;
+	public *values(recursive = false): IterableIterator<SchemaFolder | SchemaEntry> {
+		if (recursive) {
+			for (const value of super.values()) {
+				if (value.type === 'Folder') yield* (value as SchemaFolder).values(true);
+				else yield value;
+			}
+		} else {
+			yield* super.values();
+		}
+	}
+
+	/**
+	 * Returns a new Iterator object that contains the `[key, value]` pairs for each element contained in this folder and children folders.
+	 * Identical to [Map.entries()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/entries)
+	 * @param recursive Whether the iteration should be recursive
+	 */
+	public entries(recursive: true): IterableIterator<[string, SchemaEntry]>;
+	/**
+	 * Returns a new Iterator object that contains the `[key, value]` pairs for each element contained in this folder.
+	 * Identical to [Map.entries()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/entries)
+	 * @param recursive Whether the iteration should be recursive
+	 */
+	public entries(recursive?: false): IterableIterator<[string, SchemaFolder | SchemaEntry]>;
+	public *entries(recursive = false): IterableIterator<[string, SchemaFolder | SchemaEntry]> {
+		if (recursive) {
+			for (const [key, value] of super.entries()) {
+				if (value.type === 'Folder') yield* (value as SchemaFolder).entries(recursive);
```

Comment:
```suggestion
				if (value.type === 'Folder') yield* (value as SchemaFolder).entries(true);
```
To keep it consistent with all the recursive methods.